README.md

```diff
@@ -81,6 +81,38 @@ module Api
               info: "the time of this Widget's last update"
         field :created_at, :datetime,
               info: "the time at which this Widget was created"
+
+        # Fields can be nested under non-executable parent fields where the nested fields
```

Comment:
Might need a better word to explain this but `non-executable` means that the parent block is not evaluated and it's nested fields are evaluated with the presented model rather than the value from the parent block